app/src/main/java/com/geekskool/dhobi/Util.java

```diff
@@ -0,0 +1,11 @@
+package com.geekskool.dhobi;
+
+/**
+ * Created by manisharana on 22/2/17.
+ */
+public class Util {
+    public static boolean isInvalid(String name) {
```

Comment:
Since this is a validation function, you can create a `com.geekskool.dhobi.validation` package ...even if it only has one fn in it for now. :) Then rename the fn something more descriptive, like `isNameValid`.

Always prefer the positive: `isAgeValid` over `isAgeInvalid` -- `isAgeInvalid` implies a double-negative, which is harder to read.